### DISCUSSION TITLE
Welcome to maxar-open-data Discussions!

### DISCUSSION THREAD

**Discussion by giswqs** [opengeos/maxar-open-data#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sami8501** [comment]
Hello, 
Thank you very much for this awesome work.
I wanted to execute your python code, more particularly the case of Morocco as follows:

m = leafmap.Map(height='600px')
m.add_basemap('SATELLITE')
url = 'https://open.gishub.org/maxar-open-data/datasets/Morocco-Earthquake-Sept-2023/10300100ECC53700.json'
m.add_stac_layer(url, name="Mosaic")
m.add_gdf(gdf, layer_name="Footprints", info_mode="on_click")
m



The problem is that I encounter an error that I cannot overcome:

KeyError Traceback (most recent call last)
Cell In[16], line 4
      2 m.add_basemap('SATELLITE')
      3 url = 'https://open.gishub.org/maxar-open-data/datasets/Morocco-Earthquake-Sept-2023/10300100ECC53700.json'
----> 4 m.add_stac_layer(url, name="Mosaic")
      5 m.add_gdf(gdf, layer_name="Footprints", info_mode="on_click")
      6m

File ~\anaconda3\envs\geo\Lib\site-packages\leafmap\foliumap.py:1021, in Map.add_stac_layer(self, url, collection, item, assets, bands, titiler_endpoint, name, attribution, opacity, shown, fit_bounds, **kwargs)
    991 def add_stac_layer(
    992 self,
    993 url: str = None,
   (...)
   1004 **kwargs,
   1005):
   1006 """Adds a STAC TileLayer to the map.
   1007
   1008 Args:
   (...)
   1019 fit_bounds (bool, optional): A flag indicating whether the map should be zoomed to the layer extent. Defaults to True.
   1020 """
-> 1021 tile_url = stac_tile(
   1022 url, collection, item, assets, bands, titiler_endpoint, **kwargs
   1023)
   1024 bounds = stac_bounds(url, collection, item, titiler_endpoint)
   1025 self.add_tile_layer(
   1026 url=tile_url,
   1027 name=name,
   (...)
   1030 shown=shown,
   1031)

File ~\anaconda3\envs\geo\Lib\site-packages\leafmap\stac.py:704, in stac_tile(url, collection, item, assets, bands, titiler_endpoint, **kwargs)
    701 else:
    702 r = requests.get(titiler_endpoint.url_for_stac_item(), params=kwargs).json()
--> 704 return r["tiles"][0]

KeyError: 'tiles'



Can you help me solve this problem.
thank you in advance.

**Reply by giswqs** [reply]
Try updating leafmap using `pip install -U leafmap`. It works fine on my end. 

![image](https://github.com/opengeos/maxar-open-data/assets/5016453/3917f1c9-5ebf-489f-bf35-a69f3a65f713)

**Reply by sami8501** [reply]
perfect, thank you very much, it works after installing the latest version of leafmap. Sincerely
